src/Fixer/Casing/NativeTypeDeclarationCasingFixer.php

```diff
@@ -119,15 +119,15 @@ final class NativeTypeDeclarationCasingFixer extends AbstractFixer
     private FunctionsAnalyzer $functionsAnalyzer;
 
     /**
-     * @var list<list<int>>
+     * @var list<list<int>|string>
      */
-    private array $propertyTypeModifiers;
+    private array $beforePropertyTypeTokens;
 
     public function __construct()
     {
         parent::__construct();
 
-        $this->propertyTypeModifiers = [[T_PRIVATE], [T_PROTECTED], [T_PUBLIC], [T_VAR]];
+        $this->beforePropertyTypeTokens = ['{', ';', [T_PRIVATE], [T_PROTECTED], [T_PUBLIC], [T_VAR]];
```

Comment:
Shouldn't we add support for PHP8 attributes too? I know this is a fix PR, but maybe you would like to add it too 😉.

Reply:
Nevermind, auto-merge enabled 😆.

Reply:
Is it possible to add an attribute without visibility?

Reply:
[It looks like it's possible](https://3v4l.org/FLOZP#v8.0.30) 🫣.

Reply:
Will be working well, I guess we cannot make any failing test, and only add some to prevent regression.